### PR TITLE
Ref Guide - Channel Details

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10598

@keichwa I will edit the channels.adoc file in the client config separately!

### CODE REVIEW

File: modules/reference/pages/software/channel-details.adoc [hunk 1]
```diff
@@ -1,97 +1,70 @@
 [[s4-sm-channel-details-details]]
 = Channel Details
 
-General information about the channel and its parent if applicable.
-This summary, description, and architecture is also displayed when clicking a channel.
+The menu:Software[Channel List > Channel Details] section displays the details of a selected patch.
```
Comment (keichwa) [inline modules/reference/pages/software/channel-details.adoc:4]: I'm wondering whether such menu paths are ok.  It is kind of an abbreviation. You cannot click a widget with the name "Channel Details".
```suggestion
The menu:Software[Channel List > Channel Details] section displays the details of a selected channel.
```
copypasta? :wink:

File: modules/reference/pages/software/channel-details.adoc [hunk 2]
```diff
@@ -1,97 +1,70 @@
...
+== Details
 
-[NOTE]
-====
-The assigned base channel must match the installed system.
-For example, a system running {sle}{nbsp}11 for `x86_64` cannot be registered to a {sle}{nbsp}12 for `s390x` base channel.
-Use the files [path]``/etc/os-release`` or [path]``/etc/SuSE-release`` to check your product, architecture (try [command]``uname -a``), version, and patch level.
-====
+The [guimenu]``Details`` tab shows the basic channel details, including a description of the channel, and the dates it was last modified and built.
+This tab also provides contact information for the maintainer of the product and the GPG key details, where available.
```
Comment (keichwa) [inline modules/reference/pages/software/channel-details.adoc:14]: "if available"?

File: modules/reference/pages/software/channel-details.adoc [hunk 3]
```diff
@@ -1,97 +1,70 @@
...
 
 
 
-[[s4-sm-channel-details-managers]]
 == Managers
 
-On the [guimenu]``Managers`` page, you can check which users are authorized to manage the selected channel.
+The [guimenu]``Managers`` tab shows which users are authorized to manage the selected channel.
+The list shows the username, real name and email address of the channel manager, as well as the current status of the user.
```
Comment (keichwa) [inline modules/reference/pages/software/channel-details.adoc:21]:
```suggestion
The list shows the username, real name, and email address of the channel manager, as well as the current status of the user.
```

File: modules/reference/pages/software/channel-details.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/reference/pages/software/channel-details.adoc:4]: yup

File: modules/reference/pages/software/channel-details.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/reference/pages/software/channel-details.adoc:14]: either works.

Comment (keichwa) [reply modules/reference/pages/software/channel-details.adoc:14]: ok

**Pull request merged**